templates/advantage/index.html

```diff
@@ -212,6 +212,57 @@ <h2>Your paid subscriptions</h2>
                                       <span>Renewal in progress...</span>
                                     </td>
                                   </tr>
+                                {% elif contract["renewal"]["actionable"] == false %}
+                                  </table>
+                                  
+                                  <table class="u-no-margin--bottom">
```

Comment:
Why is this notification in a table?

Reply:
I think because I was tired and surrounded by other tables... I'll update this.